validator/client/src/test/java/tech/pegasys/teku/validator/client/duties/AttestationProductionDutyTest.java

```diff
@@ -237,6 +257,11 @@ public void shouldPublishProducedAttestationsWhenSignerFailsForSomeAttestations(
         .dutyFailed(
             TYPE, SLOT, Set.of(validator1.getPublicKey().toAbbreviatedString()), signingFailure);
     verifyNoMoreInteractions(validatorLogger);
+
+    verify(validatorDutyMetrics)
+        .record(any(), any(AttestationProductionDuty.class), eq(Step.CREATE));
+    verify(validatorDutyMetrics, times(2))
+        .record(any(), any(AttestationProductionDuty.class), eq(Step.SIGN));
```

Comment:
This is correct. In this test, we have two validators in the same committee, so we produce the same attestation for both of them. But they sign the attestation individually. That's why we have 1 create and 2 sign.